Makefile.core.mk

```diff
@@ -336,9 +336,7 @@ lint-helm-global:
 	find manifests -name 'Chart.yaml' -print0 | ${XARGS} -L 1 dirname | xargs -r helm lint --strict -f manifests/global.yaml
 
 lint: lint-python lint-copyright-banner lint-scripts lint-go lint-dockerfiles lint-markdown lint-yaml lint-licenses lint-helm-global
-	@bin/check_helm.sh
 	@bin/check_samples.sh
-	@bin/check_dashboards.sh
```

Comment:
Do we have an equivalent with the new charts ?

Reply:
To the best of my knowledge this is some legacy thing that no longer applies